channels/applicationinsights-channel-js/Tests/Unit/src/StatsBeat.tests.ts

```diff
@@ -19,15 +19,14 @@ export class StatsbeatTests extends AITestClass {
         }
         this._sender = null;
         this._core = null;
-
     }
 
     public registerTests() {
         this.testCase({
-            name: "Statsbeat initializes when disableStatsBeat is false",
+            name: "Statsbeat initializes when intStats is true",
             test: () => {
                 let config = {
-                    disableStatsBeat: false,
+                    _sdk: {intStats: true},
```

Comment:
as `_sdk` represents just the settings for the sdk (ie. internal), then we can set this name just to `stats`